Userland/Applications/PixelPaint/MainWidget.cpp

```diff
@@ -23,10 +23,12 @@
 #include <LibCore/MimeData.h>
 #include <LibGUI/Application.h>
 #include <LibGUI/Clipboard.h>
+#include <LibGUI/GroupBox.h>
```

Comment:
We could get away with only using forward declarations for these if we added a key function to these classes, so that the vtable (and thus the destructor) does not need to be emitted at every use.